bukuserver/static/bukuserver/css/statistic.css

```diff
@@ -0,0 +1,16 @@
+/* limit dialog height with a scrollbox */
+.modal-body {
+    max-height: calc(100vh - 120px);
+    overflow: auto;
+}
+
+/* make the table header sticky */
+.modal-body thead {
+    position: sticky;
+    /* setting `top:` with JS, to account for theme differences */
+}
+
+/* overriding icon-button text color with theme color */
+.modal-header :is(button, button:hover) {
+    color: inherit;
+}
```

Comment:
Preventing Stats dialog screen overflow (replacing it with a scrollbox), making the table header sticky, and ensuring the close button is visible regardless of UI theme
![stats dialog](https://github.com/user-attachments/assets/83234f62-ea1d-4f11-87b2-70c28724ec43)